python/eos/tasks.py

```diff
@@ -394,7 +394,7 @@ def predict_observables(analysis_file:str, posterior:str, prediction:str, base_d
     :type base_directory: str, optional
     :param begin: The index of the first sample to use for the predictions. Defaults to 0.
     :type begin: int
-    :param end: The index beyond the last sample to use for the predictions. Defaults to -1.
+    :param end: The index of the last sample to use for the predictions. Defaults to -1.
```

Comment:
Hm, I think this is wrong. Take a list ``l = ['a', 'b', 'c']``. Using ``l[:-1]`` yields ``['a', 'b']``. So the description was correct, but the default value is not. The default value should be ``None``.